h/views/api/config.py

```diff
@@ -69,6 +89,13 @@ def add_api_view(
         registry.api_links.append(link)
 
     config.add_view(view=view, **settings)
+
+    for version in versions:
```

Comment:
Should we have some tests for this?

Reply:
I so appreciate your mutation testing! Yes, I should add some unit tests for this. Note that the functional tests do fail if you comment this out. I had been "putting off" writing unit tests til my next PR, which will extend this stuff, but I'll do it now.